requirements.txt

```diff
@@ -30,6 +30,8 @@ mercurial==2.7.1
 honcho==0.5.0
 # sha256: th84eGr2uNZEtTBVIBLWacdBX89YUt4yDykIANuXmD0
 gunicorn==18.0
+# sha256: 02C9MYmPnfWfb6p4ZVEGW7pFs15-48ObOBtPv-9zkvQ
+South==1.0.2
```

Comment:
This is the cause of the Travis failure on this PR btw:
https://travis-ci.org/mozilla/socorro/builds/52057223#L163